menelaus/experimental/alarm.py

```diff
@@ -0,0 +1,97 @@
+import numpy as np
+from abc import ABC, abstractmethod
+from scipy.stats import ks_2samp
+
+
+STATE_INIT = "baseline"
```

Comment:
This setup is a good idea. Even if we didn't swap the main detector class completely over, this is cleaner than the current direct string assignment.